druid/src/env.rs

```diff
@@ -92,11 +92,8 @@ pub struct Key<T> {
     value_type: PhantomData<T>,
 }
 
-// we could do some serious deriving here: the set of types that can be stored
-// could be defined per-app
-// Also consider Box<Any> (though this would also impact debug).
 /// A dynamic type representing all values that can be stored in an environment.
-#[derive(Clone, Data, PartialEq)]
```

Comment:
This removes `PartialEq`. it is not straight forward to implement with `Arc<dyn Any>`. I think Data should be sufficient